### DISCUSSION TITLE
Why the search results are different

### DISCUSSION THREAD

**Discussion by smehtaAA** [moov-io/watchman#383] [General]
Hi,

I have been running some tests to figure out the search results and what to expect. I tried for a single word "Nicolas" in watchman. 

![image](https://user-images.githubusercontent.com/6786820/148909470-d6f40f93-2334-4239-8908-815e72ac4088.png)

The same search run on the OFAC sanctions search is different. 

![image](https://user-images.githubusercontent.com/6786820/148909669-68fac64b-b35f-45a7-8caa-607f3bd2cd31.png)

What can be reason for this? Is there a recommended search option to get accurate results?

Thanks,

**Comment by adamdecaf** [comment]
`EXACT_MATCH_FAVORITISM` is included if you want to match the Treasury tool. Their search produces tons of false positives so we have purposefully designed Watchman to return better match rankings. 

`EXACT_MATCH_FAVORITISM=1.0` would produce results similar to the Treasury OFAC search tool.

**Reply by smehtaAA** [reply]
Thanks. I will check it out. Also, does it affect the results which are not an exact match?

**Reply by adamdecaf** [reply]
> does it affect the results which are not an exact match?

Nope. The match has to be exact. https://github.com/moov-io/watchman/blob/v0.21.3/cmd/server/search.go#L848-L850
